pkg/cli/trcconfigbase/trcconfig.go

```diff
@@ -172,6 +173,9 @@ func CommonMain(envPtr *string,
 			*wantCertsPtr = true
 		}
 	}
+	if envPtr == nil {
```

Comment:
We probably want to do `flagset.String("env", envDefaultPtr ...)` here, as wouldn't `envPtr` never be null further down this logic?